src/DistributedWriteAheadLog/Record.h

```diff
@@ -51,7 +52,9 @@ struct Record
         return OpCode::UNKNOWN;
     }
 
-    static ByteVector write(const Record & record);
+    static uint8_t compression(uint64_t flags) { return (flags >> 11ul) & 0x1F; }
```

Comment:
Let's prefix `ALWAYS_INLINE` for `opcode`, `version` and `compressed` functions.

`compression` -> `compressed` if we don't want support different compression algorithms